src/index.ts

```diff
@@ -57,114 +56,238 @@ export { default as GlintsContainer } from './Style/GlintsContainerStyle';
 
 // Icons
 export { default as AddIcon } from './General/Icon/components/AddIcon';
-export { default as AddCircleOutlineIcon } from './General/Icon/components/AddCircleOutlineIcon';
-export { default as AddCircleSolidIcon } from './General/Icon/components/AddCircleSolidIcon';
-export { default as ArrowBackIcon } from './General/Icon/components/ArrowBackIcon';
-export { default as ArrowBackDoubleIcon } from './General/Icon/components/ArrowBackDoubleIcon';
-export { default as ArrowDownIcon } from './General/Icon/components/ArrowDownIcon';
-export { default as ArrowDownCircleOutlineIcon } from './General/Icon/components/ArrowDownCircleOutlineIcon';
-export { default as ArrowDownSolidIcon } from './General/Icon/components/ArrowDownSolidIcon';
-export { default as ArrowUpSolidIcon } from './General/Icon/components/ArrowUpSolidIcon';
-export { default as ArrowNextIcon } from './General/Icon/components/ArrowNextIcon';
-export { default as ArrowNextDoubleIcon } from './General/Icon/components/ArrowNextDoubleIcon';
-export { default as ArrowNextSolidIcon } from './General/Icon/components/ArrowNextSolidIcon';
+export {
+  default as AddCircleOutlineIcon,
+} from './General/Icon/components/AddCircleOutlineIcon';
```

Comment:
Seem to those code changes were made by prettier. Could you help to revert them via command `yarn storybook`. So that another new PR related to this file won't have those lines of code change again.